api/node/src/interpreter/diagnostic.rs

```diff
@@ -35,14 +35,14 @@ pub struct JsDiagnostic {
     /// Message for this diagnostic.
     pub message: String,
 
-    /// The line number in the .slint source file.
+    /// The line number in the .slint source file. The line number start with 1.
```

Comment:
"start" -> "starts" here and below for the column.

Reply:
done